### PR TITLE
Update preset-security-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Added PS cmdlets to review the settings

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9195

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review.

**Comment by msbemba** [comment]
@yogkumgit The Author is chrisda

File: microsoft-365/security/office-365-security/preset-security-policies.md [hunk 1]
```diff
@@ -223,6 +223,23 @@ The steps to modify the assignment of the **Standard protection** or **Strict pr
 
 To disable the **Standard protection** or **Strict protection** preset security policies while still preserving the existing conditions and exceptions, slide the toggle to **Disabled** ![Toggle Off.](../../media/scc-toggle-off.png). To enable the policies, slide the toggle to **Enabled** ![Toggle On](../../media/scc-toggle-on.png).
 
+### Use PowerShell to review the settings of the preset security policy
+
+There are two cmdlets that we can use to view these policies:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/preset-security-policies.md]:
```suggestion
There are two cmdlets that you can use to view these policies:
```

File: microsoft-365/security/office-365-security/preset-security-policies.md [hunk 2]
```diff
@@ -223,6 +223,23 @@ The steps to modify the assignment of the **Standard protection** or **Strict pr
...
+
+1. To View Safe Attachments and Safe Link Policies, you can use the below Cmdlet
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/preset-security-policies.md]:
```suggestion
1. To view Safe Attachments and Safe Link policies, use the following cmdlet:
```

File: microsoft-365/security/office-365-security/preset-security-policies.md [hunk 3]
```diff
@@ -223,6 +223,23 @@ The steps to modify the assignment of the **Standard protection** or **Strict pr
...
+
+```
+Get-ATPProtectionPolicyRule
+```
+
+2. To View Hosted Connection Filter, Anti-Phish and Malware Filter you can use the below Cmdlet
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/preset-security-policies.md]:
```suggestion
2. To view the Hosted Connection, Anti-Phish, and Malware filters, use the following cmdlet:
```

File: microsoft-365/security/office-365-security/preset-security-policies.md [hunk 4]
```diff
@@ -223,6 +223,23 @@ The steps to modify the assignment of the **Standard protection** or **Strict pr
...
+
+```
+Get-EOPProtectionPolicyRule
+```
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/preset-security-policies.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by chrisda** [comment]
I was in discussion about publishing these \*-\*ProtectionPolicyRule cmdlet reference topics way back in 2020 when preset security policies went live. I have them all semi-documented in a branch in the office-docs-powershell repo, but I never published them on advice from the PM.

A preset security policy simply creates a bunch of regular polices (EOP: malware, spam, phishing (spoof); MDO: phishing (impersonation) Safe Links, Safe Attachments) with names like 'Standard Preset Security Policy1622650008534' or 'Strict Preset Security Policy1642034873192' you can view the settings using the regular Get- cmdlets (i.e., Get-MalwareFilterPolicy).

The \*-EOPProtectionPolicyRule and \*-ATPProtectionPolicyRule cmdlets link the rules to the Standard or Strict polices, and determine who the policies apply to. For some reason, preset security policies couldn't use the regular \*-\<Feature\>Rule cmdlets that are normally associated with policies (i.e, \*-MalwareFilterRule for \*-MalwareFilterPolicy).

So, I don't want to approve this PR (yet) because the cmdlets you added to this topic aren't present in the PowerShell repo. I need to ponder the best way forward.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrisda** [comment]
Closing this PR as a massive, better version is coming

**Pull request closed without merging**